lib/config.js

```diff
@@ -31,6 +31,7 @@ const paramsSchema = Joi.object().keys({
   errorOnRequiredAuth: Joi.boolean().optional().default(false),
   auth0Logout: Joi.boolean().optional().default(false),
   redirectUriPath: Joi.string().optional().default('/callback'),
+  post_logout_redirect_uri: Joi.string().optional(),
```

Comment:
Only used `post_logout_redirect_uri` instead of `postLogoutRedirectUri`, cause it is how it is referred to here:
https://github.com/auth0/express-openid-connect/blob/7050ffcd29470a396dbb2883c86349fb1ae24092/lib/context.js#L118
and here:
https://github.com/auth0/express-openid-connect/blob/7050ffcd29470a396dbb2883c86349fb1ae24092/lib/client.js#L57

But no problem to fix it. (I see I can also include it in the `index.d.ts` if we agree on the name.)

Also, not sure how to tell Joi to use `baseURL` as default:
`postLogoutRedirectUri: Joi.string().optional().default(????),`

And where to actually say that the param should default to baseURL. Somewhere in `buildAuthorizeParams`?

Reply:
I see what you're saying. Maybe the better way to go about this is to follow the example of `redirectUriPath` and set that as a path, instead of a complete URI, and append the base URL when we use it. That would be more consistent and make a bit more sense as that should not be an external URL.